Tests/FunctionalTests/XCTestManifests.swift

```diff
@@ -27,32 +27,6 @@ extension DependencyResolutionTests {
     ]
 }
 
-extension MiscellaneousTestCase {
```

Comment:
This diff and removal of the `MiscellaneousTests.swift` file seems like a mistake? Can we revert that?